### DISCUSSION TITLE
Field Order In Layout

### DISCUSSION THREAD

**Discussion by mcqueena9845** [directus/directus#3455] [Feature Requests]
The order of the fields in the layout sidebar for non-selected fields seems random.  It's same for other field selection drop-downs.  They should be alphabetical or the order they were added in the collection.

![image](https://user-images.githubusercontent.com/75855426/102543847-e319a680-4081-11eb-9532-835a0635089a.png)

**Comment by benhaynes** [comment]
Haha, the first Discussion that I wish was an Issue! 😄

**Reply by mcqueena9845** [reply]
I already opened 3 issues today, so I thought I would make this a discussion.  Should I move it?

**Reply by benhaynes** [reply]
Haha, usually I convert issues (that are really questions) to Discussions... but yeah, I think this is something we'll definitely want to do, so let's make it as an issue!

**Reply by mcqueena9845** [reply]
Done

**Comment by rijkvanzanten** [comment]
This has since been addressed
